Changelog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## UNRELEASED
+
+### API
+- New `coq.reduction.lazy.norm`
+- New `coq.reduction.native.norm`
+- New `coq.reduction.native.available?`
+- Rename `coq.reduction.cbv.whd_all` -> `coq.reduction.cbv.norm`
```

Comment:
According to the Coq reference manual, the normalization process of `cbv` recursively applies weak-head normal form computation as in `lazy` (I guess the difference is subterms under lambda abstractions). So you probably want to keep `whd_all` here but not so sure.

Reply:
For lazy, I could find entry points for weak head normal form, and full normal form. I could not for cbv, which seems to be always recursing under binders. So my "deprecation" is really giving the right name to the code which was already there, for cbv (unless I got mistaken in understanding the Coq API I'm calling)

Reply:
OK. The reference manual may have to be fixed then.

Reply:
I don't think that the `lazy.whd_all`  procedure I bind in coq-elpi is available to the Coq user, so the manual is right, since `Eval lazy` or `lazy` do a full normalization. *but* the kernel of Coq uses "the base case of lazy" for type checking, so the API is there and I bind it as `whd_all` since I believe it is useful.

Reply:
What I was saying is that I was wrong in thinking that the base case of vm and cbv was there, so I bound them with a `.whd_all` suffix, but I was wrong, since they do `.norm`.

Reply:
For me, the problem is that the quoted paragraph seems to apply to both `lazy` and `cbv` but it is not true for the latter, IIUC.

Reply:
> Once a weak-head normal form is obtained, subterms are recursively reduced using the same strategy.

This means, IMO, that you obtain a full normal form

Reply:
I agree with that point, but my understanding is that there is no "weak-head normal form" computation (that is different from the full normal form) for `cbv`. So, that sentence does not quite reflect the actual normalization procedure.

In any case, the changes this PR proposes are sensible to me.